app/router.ts

```diff
@@ -25,10 +25,16 @@ Router.map(function() {
         this.route('discover', { path: '/:institution_id' });
         this.route('dashboard', { path: '/:institution_id/dashboard' });
     });
+
     this.route('preprints', function() {
-        this.route('discover');
+        this.route('index', { path: '/' });
+        this.route('index', { path: '/:provider_id' });
         this.route('discover', { path: '/:provider_id/discover' });
+        // this.route('detail', { path: '--preprint/:guid' });
+        this.route('detail', { path: '../:guid' });
```

Comment:
I have two questions about this route definition:
1. Why `../:guid` instead of `/:guid`?
2. What happens if the preprint is on a custom provider? Does the url show `osf.io/preprints/<preprint_guid>` or `osf.io/<provider_id>/<preprint_guid>`?